.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+python:
+  - 3.4
+  - 3.5
```

Comment:
not sure, you may need to give it as a string: https://docs.travis-ci.com/user/languages/python/#Default-Python-Version